browser/ui/views/speedreader/reader_mode_toolbar_view.cc

```diff
@@ -72,5 +129,18 @@ void ReaderModeToolbarView::OnBoundsChanged(const gfx::Rect& previous_bounds) {
   toolbar_->SetBoundsRect(toolbar_bounds);
 }
 
+bool ReaderModeToolbarView::OnMousePressed(const ui::MouseEvent& event) {
+  if (event.IsOnlyLeftMouseButton() && delegate_) {
+    delegate_->OnReaderModeToolbarActivate(this);
+  }
+  return views::View::OnMousePressed(event);
+}
+
+void ReaderModeToolbarView::ActivateContents(content::WebContents* contents) {
```

Comment:
Confirmed that this is only called on Windows. See `WebContentsViewAura::OnMouseEvent()`.
Trying to find another way but no luck yet..